packages/SystemsThinkingWorkbench-Core.package/STWPlayfield.class/instance/addSimulationButtons.st

```diff
@@ -0,0 +1,15 @@
+initialization
+addSimulationButtons
+	
+	self playButton: STWButton new.
+	self playButton contents: 'Start'.
+	self playButton clickHandler: [ self toggleSimulation. ].
+	self addMorph: self playButton.
+	self playButton position: self position.
+	
+	self stopButton: STWButton new.
+	self stopButton contents: 'Reset'.
+	self addMorph: self stopButton.
+	self stopButton clickHandler: [self resetSimulation. ].
+	self stopButton position: self position + ((self playButton extent x + 10)@ 0).
```

Comment:
Actually, that is a reset button, which stops the simulation and resets it to the initial state, will rename.